test/Suites.scala

```diff
@@ -0,0 +1,7 @@
+import integration.IntegrationTests
+import org.scalatest.Suites
+import test.ThriftUtilSpec
+
+class UnitTestsSuite extends Suites(new ThriftUtilSpec)
```

Comment:
I've done that before by putting integration tests in `src/it/scala` and adding that as a test root but it always felt smelly and non-standard